App/ViewModels/SignInViewModel.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+
+namespace Coder.Desktop.App.ViewModels;
+
+/// <summary>
+/// The View Model backing the sign in window and all its associated pages.
+/// </summary>
+public partial class SignInViewModel : ObservableObject
+{
+    public SignInViewModel()
+    {
+        _url = string.Empty;
+        CoderToken = string.Empty;
+    }
+
+    [ObservableProperty]
+    private string _url;
```

Comment:
Try this instead to avoid the warnings:
```
[ObservableProperty]
public partial Type Name { get; set; } = DefaultValue;
```

If you want the setter to be private, use `private set;`. And since they're now public they should have capitalized named.